plugins/org.python.pydev/src/org/python/pydev/editor/actions/PyComment.java

```diff
@@ -82,11 +83,15 @@ public void run(IAction action) {
      * @return the new selection
      * @throws BadLocationException
      */
-    protected Tuple<Integer, Integer> performComment(TextSelectionUtils ps, boolean addCommentsAtIndent)
+    protected Tuple<Integer, Integer> performComment(TextSelectionUtils ps, String addCommentsOption)
             throws BadLocationException {
-        LineCommentAction lineCommentAction = new LineCommentAction(ps, "#", this.std.spacesInStartComment,
-                addCommentsAtIndent);
+        if (addCommentsOption == null
+                || !CommentBlocksPreferences.getValuesForAddCommentsOption().contains(addCommentsOption)) {
+            org.python.pydev.core.log.Log.log("Not expected add comments option.");
```

Comment:
Message could be:

`"Unexpected option to add comments: " + addCommentsOption`